docs/introduction.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Introduction
+sidebar_position: 0
+---
+
+
+# Introduction
+
+:::warning
+Check out our [Quick Start](./quickstart) guide to get started in a snap.
+:::
+
+Blindnet offers a simple solution to enable data privacy and compliance with data minimization requirements in various data regulations, including GDPR and CCPA / CPRA. Beyond compliance, we provide software developers with a zero-trust service to minimize the exposure to unnecessary user data and reduce the risk in case of data breaches, via two complementary solutions:
```

Comment:
```suggestion
Blindnet offers a simple solution to enable data privacy and compliance with requirements in various data regulations, including GDPR and CCPA. 
Beyond compliance, we provide software developers with a service to minimize the exposure to unnecessary user data and reduce the risk in case of data breaches, via two complementary solutions:
```
We can't claim we support CPRA for the moment. 

We don't effectively supprot data minimization requirements, as we in fact offer nothing to help developers reduce the amount of unnecessary data that they colelct and store.

Reply:
That's right, thanks for pointing it out. This underline that we **MUST** quickly communicate about this part in blindnet devkit.